### PR TITLE
next-auth & formidable

### PR DESCRIPTION
**Pull request by taoqf**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <https://next-auth.js.org/providers/email>
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

File: types/formidable/index.d.ts [hunk 1]
```diff
@@ -34,7 +34,7 @@ export interface Fields {
 }
 
 export interface Files {
-    [key: string]: File; // | File[];
+    [key: string]: File | File[];	// is an array when multiples is true
```
Comment (danger-public) [inline types/formidable/index.d.ts:37]: - :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.

**Comment by danger-public** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## formidable ([<kbd>unpkg</kbd>](https://unpkg.com/browse/formidable@latest/))
was missing the following properties:
1. The declaration doesn't match the JavaScript module 'formidable'. Reason:
The JavaScript module can be called or constructed, but the declaration module cannot.

The most common way to resolve this error is to use 'export =' syntax.
To learn more about 'export =' syntax, see https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 388289189cf15962f95670a4cef0744444c2174a
</p>

**Comment by JuanM04** [comment]
Well, thank you for your contribution to `next-auth`. Now, @Nemo157 has to check your changes in `formidable`.

**Comment by orta** [comment]
Poke @Nemo157  - its a small breaking change (as evidenced by the tests change) but I'd rather not merge it without someone who knows about the API usage at least giving some input

**Comment by orta** [comment]
OK, let's do it 👍🏻

**Pull request merged**